lib/index.js

```diff
@@ -771,6 +773,31 @@ class GalleryVideoArticle extends BaseAsset {
 
 exports.GalleryVideoArticle = GalleryVideoArticle;
 
+/**
+ * A simple wrapper over VideoAsset that should be used for toplevel videos.
+ *
+ * Along with the usual VideoAsset methods, it also supports set_author,
+ * which allows it to be used in place of GalleryVideoArticle.
+ */
+class VideoArticle extends VideoAsset {
+
+    /**
+     * Set the author of this document to the given value.
```

Comment:
More editorial comments: it would be a bonus to mention how "author" relates to video, i.e. that the author is the creator of the video.

Reply:
Yup, fair enough.